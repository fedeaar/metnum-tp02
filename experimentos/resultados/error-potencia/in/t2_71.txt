7.826271000000000000e+06 3.804668000000000000e+06 1.347117000000000000e+06 -1.141047000000000000e+06 -2.482110000000000000e+05 3.255910000000000000e+05 3.017160000000000000e+05 9.015670000000000000e+05 -1.783230000000000000e+06 -1.454982000000000000e+06 -1.140160000000000000e+06 -7.735200000000000000e+05 1.237432000000000000e+06 3.947388000000000000e+06 1.788729000000000000e+06 6.732290000000000000e+05 -1.661955000000000000e+06 2.489072000000000000e+06 2.300913000000000000e+06 -1.688730000000000000e+05
3.804668000000000000e+06 6.946047000000000000e+06 4.816990000000000000e+05 -1.943536000000000000e+06 1.004296000000000000e+06 1.153001000000000000e+06 -2.600970000000000000e+06 8.129080000000000000e+05 -5.891550000000000000e+05 -8.831330000000000000e+05 -2.963600000000000000e+05 -1.799387000000000000e+06 2.283875000000000000e+06 1.348156000000000000e+06 1.436058000000000000e+06 4.774700000000000000e+05 -7.971080000000000000e+05 -9.187300000000000000e+05 1.215680000000000000e+06 -7.283750000000000000e+05
1.347117000000000000e+06 4.816990000000000000e+05 5.038993000000000000e+06 1.870953000000000000e+06 6.084710000000000000e+05 1.027733000000000000e+06 -9.590020000000000000e+05 -1.369600000000000000e+04 4.824220000000000000e+05 4.733540000000000000e+05 6.896580000000000000e+05 -3.210160000000000000e+05 -9.713560000000000000e+05 4.750640000000000000e+05 -1.735017000000000000e+06 -1.168428000000000000e+06 -1.499750000000000000e+06 1.045640000000000000e+06 1.873689000000000000e+06 -3.641940000000000000e+05
-1.141047000000000000e+06 -1.943536000000000000e+06 1.870953000000000000e+06 6.821397000000000000e+06 -3.591530000000000000e+05 -7.375170000000000000e+05 2.297490000000000000e+05 7.520960000000000000e+05 -5.682710000000000000e+05 -5.511730000000000000e+05 8.266130000000000000e+05 8.257380000000000000e+05 2.710350000000000000e+05 5.951270000000000000e+05 -2.658269000000000000e+06 5.463770000000000000e+05 -3.124850000000000000e+05 -5.633940000000000000e+05 -1.042882000000000000e+06 -1.367951000000000000e+06
-2.482110000000000000e+05 1.004296000000000000e+06 6.084710000000000000e+05 -3.591530000000000000e+05 6.667611000000000000e+06 2.721501000000000000e+06 4.053300000000000000e+04 1.384685000000000000e+06 4.600579000000000000e+06 3.193000000000000000e+06 1.285578000000000000e+06 -2.176063000000000000e+06 7.932680000000000000e+05 -3.324080000000000000e+05 8.879320000000000000e+05 -7.927090000000000000e+05 9.960170000000000000e+05 -1.131116000000000000e+06 1.006969000000000000e+06 -1.457958000000000000e+06
3.255910000000000000e+05 1.153001000000000000e+06 1.027733000000000000e+06 -7.375170000000000000e+05 2.721501000000000000e+06 6.838493000000000000e+06 -1.467079000000000000e+06 -8.034720000000000000e+05 2.328038000000000000e+06 -8.020740000000000000e+05 2.960160000000000000e+06 -5.735720000000000000e+05 -1.759167000000000000e+06 -4.766440000000000000e+05 -3.953780000000000000e+05 2.603309000000000000e+06 4.038740000000000000e+05 -1.936241000000000000e+06 3.479914000000000000e+06 -1.330856000000000000e+06
3.017160000000000000e+05 -2.600970000000000000e+06 -9.590020000000000000e+05 2.297490000000000000e+05 4.053300000000000000e+04 -1.467079000000000000e+06 4.457026000000000000e+06 -6.723460000000000000e+05 -9.916180000000000000e+05 1.783261000000000000e+06 -3.003530000000000000e+05 7.618200000000000000e+05 1.823464000000000000e+06 1.562843000000000000e+06 -8.515240000000000000e+05 -6.320020000000000000e+05 7.121160000000000000e+05 9.674040000000000000e+05 -1.330469000000000000e+06 8.821250000000000000e+05
9.015670000000000000e+05 8.129080000000000000e+05 -1.369600000000000000e+04 7.520960000000000000e+05 1.384685000000000000e+06 -8.034720000000000000e+05 -6.723460000000000000e+05 7.080439000000000000e+06 7.633290000000000000e+05 2.036538000000000000e+06 1.377886000000000000e+06 1.972490000000000000e+06 -2.194278000000000000e+06 2.657280000000000000e+05 1.891795000000000000e+06 -1.897052000000000000e+06 1.782634000000000000e+06 -2.702190000000000000e+06 8.137450000000000000e+05 -1.503445000000000000e+06
-1.783230000000000000e+06 -5.891550000000000000e+05 4.824220000000000000e+05 -5.682710000000000000e+05 4.600579000000000000e+06 2.328038000000000000e+06 -9.916180000000000000e+05 7.633290000000000000e+05 6.969639000000000000e+06 9.831840000000000000e+05 4.371570000000000000e+05 -4.135610000000000000e+05 6.939890000000000000e+05 -3.039160000000000000e+05 8.551780000000000000e+05 2.479390000000000000e+05 3.560000000000000000e+05 8.415090000000000000e+05 2.234142000000000000e+06 -2.651099000000000000e+06
-1.454982000000000000e+06 -8.831330000000000000e+05 4.733540000000000000e+05 -5.511730000000000000e+05 3.193000000000000000e+06 -8.020740000000000000e+05 1.783261000000000000e+06 2.036538000000000000e+06 9.831840000000000000e+05 6.191627000000000000e+06 1.946148000000000000e+06 -3.206360000000000000e+05 -4.464310000000000000e+05 -1.854851000000000000e+06 -7.789800000000000000e+04 -4.514412000000000000e+06 2.069373000000000000e+06 -1.539641000000000000e+06 -1.079848000000000000e+06 -1.251670000000000000e+05
-1.140160000000000000e+06 -2.963600000000000000e+05 6.896580000000000000e+05 8.266130000000000000e+05 1.285578000000000000e+06 2.960160000000000000e+06 -3.003530000000000000e+05 1.377886000000000000e+06 4.371570000000000000e+05 1.946148000000000000e+06 6.143773000000000000e+06 2.387390000000000000e+05 -2.557161000000000000e+06 -2.645962000000000000e+06 -1.290360000000000000e+05 2.385970000000000000e+05 2.534057000000000000e+06 -3.823617000000000000e+06 -9.291500000000000000e+05 6.698600000000000000e+04
-7.735200000000000000e+05 -1.799387000000000000e+06 -3.210160000000000000e+05 8.257380000000000000e+05 -2.176063000000000000e+06 -5.735720000000000000e+05 7.618200000000000000e+05 1.972490000000000000e+06 -4.135610000000000000e+05 -3.206360000000000000e+05 2.387390000000000000e+05 4.709278000000000000e+06 -1.718458000000000000e+06 1.952400000000000000e+04 -7.634050000000000000e+05 1.011439000000000000e+06 5.224750000000000000e+05 4.295830000000000000e+05 9.115000000000000000e+04 -1.103130000000000000e+05
1.237432000000000000e+06 2.283875000000000000e+06 -9.713560000000000000e+05 2.710350000000000000e+05 7.932680000000000000e+05 -1.759167000000000000e+06 1.823464000000000000e+06 -2.194278000000000000e+06 6.939890000000000000e+05 -4.464310000000000000e+05 -2.557161000000000000e+06 -1.718458000000000000e+06 8.240084000000000000e+06 3.922215000000000000e+06 -2.457240000000000000e+05 1.441900000000000000e+04 -4.547740000000000000e+05 1.954159000000000000e+06 -1.686572000000000000e+06 -1.513332000000000000e+06
3.947388000000000000e+06 1.348156000000000000e+06 4.750640000000000000e+05 5.951270000000000000e+05 -3.324080000000000000e+05 -4.766440000000000000e+05 1.562843000000000000e+06 2.657280000000000000e+05 -3.039160000000000000e+05 -1.854851000000000000e+06 -2.645962000000000000e+06 1.952400000000000000e+04 3.922215000000000000e+06 9.553394000000000000e+06 -6.892900000000000000e+04 1.078640000000000000e+05 -1.811090000000000000e+06 1.744935000000000000e+06 5.153450000000000000e+05 -2.456885000000000000e+06
1.788729000000000000e+06 1.436058000000000000e+06 -1.735017000000000000e+06 -2.658269000000000000e+06 8.879320000000000000e+05 -3.953780000000000000e+05 -8.515240000000000000e+05 1.891795000000000000e+06 8.551780000000000000e+05 -7.789800000000000000e+04 -1.290360000000000000e+05 -7.634050000000000000e+05 -2.457240000000000000e+05 -6.892900000000000000e+04 4.492891000000000000e+06 5.406700000000000000e+04 -1.156830000000000000e+05 8.626700000000000000e+04 -2.021860000000000000e+05 3.133400000000000000e+04
6.732290000000000000e+05 4.774700000000000000e+05 -1.168428000000000000e+06 5.463770000000000000e+05 -7.927090000000000000e+05 2.603309000000000000e+06 -6.320020000000000000e+05 -1.897052000000000000e+06 2.479390000000000000e+05 -4.514412000000000000e+06 2.385970000000000000e+05 1.011439000000000000e+06 1.441900000000000000e+04 1.078640000000000000e+05 5.406700000000000000e+04 8.026994000000000000e+06 -1.391608000000000000e+06 3.913380000000000000e+05 2.682410000000000000e+05 8.411280000000000000e+05
-1.661955000000000000e+06 -7.971080000000000000e+05 -1.499750000000000000e+06 -3.124850000000000000e+05 9.960170000000000000e+05 4.038740000000000000e+05 7.121160000000000000e+05 1.782634000000000000e+06 3.560000000000000000e+05 2.069373000000000000e+06 2.534057000000000000e+06 5.224750000000000000e+05 -4.547740000000000000e+05 -1.811090000000000000e+06 -1.156830000000000000e+05 -1.391608000000000000e+06 4.590190000000000000e+06 -3.331504000000000000e+06 3.226710000000000000e+05 -4.141000000000000000e+03
2.489072000000000000e+06 -9.187300000000000000e+05 1.045640000000000000e+06 -5.633940000000000000e+05 -1.131116000000000000e+06 -1.936241000000000000e+06 9.674040000000000000e+05 -2.702190000000000000e+06 8.415090000000000000e+05 -1.539641000000000000e+06 -3.823617000000000000e+06 4.295830000000000000e+05 1.954159000000000000e+06 1.744935000000000000e+06 8.626700000000000000e+04 3.913380000000000000e+05 -3.331504000000000000e+06 6.678587000000000000e+06 9.827100000000000000e+04 4.140050000000000000e+05
2.300913000000000000e+06 1.215680000000000000e+06 1.873689000000000000e+06 -1.042882000000000000e+06 1.006969000000000000e+06 3.479914000000000000e+06 -1.330469000000000000e+06 8.137450000000000000e+05 2.234142000000000000e+06 -1.079848000000000000e+06 -9.291500000000000000e+05 9.115000000000000000e+04 -1.686572000000000000e+06 5.153450000000000000e+05 -2.021860000000000000e+05 2.682410000000000000e+05 3.226710000000000000e+05 9.827100000000000000e+04 8.217688000000000000e+06 -1.751803000000000000e+06
-1.688730000000000000e+05 -7.283750000000000000e+05 -3.641940000000000000e+05 -1.367951000000000000e+06 -1.457958000000000000e+06 -1.330856000000000000e+06 8.821250000000000000e+05 -1.503445000000000000e+06 -2.651099000000000000e+06 -1.251670000000000000e+05 6.698600000000000000e+04 -1.103130000000000000e+05 -1.513332000000000000e+06 -2.456885000000000000e+06 3.133400000000000000e+04 8.411280000000000000e+05 -4.141000000000000000e+03 4.140050000000000000e+05 -1.751803000000000000e+06 3.680680000000000000e+06
