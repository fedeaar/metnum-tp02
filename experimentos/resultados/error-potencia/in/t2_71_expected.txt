2.310498873897976428e+07
1.904473177152521908e+07
1.539153449425975233e+07
1.249322068301103637e+07
1.156385921414538473e+07
1.007066691716854833e+07
8.654936391199026257e+06
6.987707068698357791e+06
6.223712922399713658e+06
4.211584793854122981e+06
3.428303063770515844e+06
2.745027667879856657e+06
1.796302711854297668e+06
1.244359033005026402e+06
8.481359374484017026e+05
6.417083080456716707e+05
4.602427969524778309e+05
2.076518909501079470e+05
5.052191125750861102e+04
1.905683595340397915e+03
